core/templates/dev/head/pages/community-dashboard-page/contributions-and-review/contributions-and-review.directive.ts

```diff
@@ -94,6 +158,94 @@ angular.module('oppia').directive('contributionsAndReview', [
                 return true;
               }));
           };
+
+          var _showQuestionSuggestionModal = function(
+              suggestion, contributionDetails, reviewable) {
+            var _templateUrl = UrlInterpolationService.getDirectiveTemplateUrl(
+              '/pages/community-dashboard-page/modal-templates/' +
+              'question-suggestion-review.directive.html');
+            var targetId = suggestion.target_id;
+            var suggestionId = suggestion.suggestion_id;
+            var authorName = suggestion.author_name;
+            var questionHeader = (contributionDetails.topic_name + ' / ' +
+              contributionDetails.skill_description);
+            var question = QuestionObjectFactory.createFromBackendDict(
+              suggestion.change.question_dict);
+            var contentHtml = question.getStateData().content.getHtml();
+
+            $uibModal.open({
+              templateUrl: _templateUrl,
+              backdrop: true,
+              size: 'lg',
+              resolve: {
+                authorName: function() {
+                  return authorName;
+                },
+                contentHtml: function() {
+                  return contentHtml;
+                },
+                question: function() {
+                  return question;
+                },
+                questionHeader: function() {
+                  return questionHeader;
+                },
+                reviewable: function() {
```

Comment:
What is this?

Reply:
Oh I meant what was the "reviewable" function? I am not too good with Angular myself, and I would defer the discussion on whether this is the right way to do it to @DubeySandeep.

Reply:
Ah I see, Thanks for the clarification!